nailgun/entities.py

```diff
@@ -3270,6 +3270,72 @@ def search_normalize(self, results):
         return super(Interface, self).search_normalize(results)
 
 
+class JobInvocation(
+        Entity,
```

Comment:
In the docs I can't find the way to Update/Delete this entity, only Create/Read. If so, please remove unneeded mixins.